scripts/provisioning/setupcmd.py

```diff
@@ -73,3 +85,27 @@ def read_ldap_credentials(self):
     except Exception as e:
       sys.stderr.write(f'unknown exception: {e}\n')
       raise e
+
+  def read_cluster_id(self):
+    """Get 'cluster>cluster_id' from confstore."""
+    prov_config_yaml = None
+
+    try:
+      with open(self.s3_prov_config) as provconf:
+        prov_config_yaml = yaml.safe_load(provconf)
```

Comment:
We have been advised to use confstore to load and read any config file. refer to my PR: https://github.com/Seagate/cortx-s3server/pull/707

Reply:
thanks addressed this point.